addons/OPT/gps/fn_clientInit.sqf

```diff
@@ -0,0 +1,530 @@
+/**
+* Spielerdarstellung auf der Karte 
+* 
+* Autor: [GNC]Lord-MDB
+*
+* Argumente:
+* kein
+*
+* Rückgabewert:
+* kein
+*
+* Server Only:
+* Nein
+* 
+* Lokal:
+* Ja
+* 
+* Global:
+* Nein
+*
+* API:
+* Nein
+* 
+* Beispiel Externer Aufruf:
+* [] call EFUNC(gps,clientInit);
+* Beispiel interner Aufruf:
+* [] call FUNC(clientInit);
+*
+*/
+
+#include "macros.hpp"
+
+DUMP("Successfully loaded the OPT/GPS module on the client");
+
+// Event Liste
+#include "\opt\opt\addons\opt\tracker\Events.hpp"
+
+// Textur Liste
+#include "\opt\opt\addons\opt\tracker\Texturen.hpp"
+
+// Offizier Classname
+GVAR(officer) = [
+    "OPT_NATO_Offizier_T",
+    "OPT_CSAT_Offizier_T",
+	"OPT_NATO_Offizier",
+	"OPT_CSAT_Offizier"
+];
+
+// Kontrolle ob Map oder Dialog geöffnet wird
+DFUNC(dialogCheck) = 
+{
+	private _verarbeitungEin = false;
+
+	//OPT Karten-Dialog
+	if (!(isNull ((findDisplay 444001) displayCtrl 10007))) then
+		{
+			_verarbeitungEin = true;
+		};
+
+	//BIS Artillery Dialog
+	if (!(isNull ((findDisplay -1) displayCtrl 500))) then
+		{
+			_verarbeitungEin = true;
+		};
+
+	//BIS  UAV Dialog	
+	if (!(isNull ((findDisplay 160) displayCtrl 51))) then
+		{
+			_verarbeitungEin = true;
+		};
+
+	//BIS  Karte und Minimap
+	if (visibleMap or visibleGPS) then
+		{
+			_verarbeitungEin = true;
+		};
+
+	_verarbeitungEin	
```

Comment:
Die Boolean-Abfragen können in einem einzigen Statement zusammen gefasst werden:

`!(isNull ((findDisplay 444001) displayCtrl 10007)) || !(isNull ((findDisplay -1) displayCtrl 500)) || !(isNull ((findDisplay 160) displayCtrl 51)) || visibleMap || visibleGPS`